### ISSUE TITLE
Playlist duplicating when dragged in the web app

### ISSUE THREAD

**Issue by urbainn** [riverscuomo/public-bug-hunt#4]
**Describe the bug**
Dragging a playlist from the navigation bar will visually duplicate it in the web app. Even when dragged on themselves. Most of the time, the playlist will replace the one on top of it, but it may also create a new element if done enough times.

**To Reproduce**
1. Go to the web app
2. On the left navigation bar, start dragging a playlist/element
3. Release the playlist
4. Playlist should have been duplicated

**Expected behavior**
I would have expected the playlist to move according to how you drag it, but not duplicate.

**Screenshots**
![dldldlsdlsdlssdfssdfsdfsdfsbgsddeLA PIERRREddddddddmosdfsLA MINEdfdcemo](https://github.com/riverscuomo/public-bug-hunt/assets/47057465/774f168b-b30d-46e7-b90a-648e3782148e)

![image](https://github.com/riverscuomo/public-bug-hunt/assets/47057465/a3534b2e-ffde-4c8c-a6b2-c7d3db39518b)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 116.0.5845.141 (Official build) (64 bits)

**Smartphone (please complete the following information):**
N/A

**Additional context**
None.

**Comment by SlightlyCompletely** [comment]
Also occurs in Weezify 1.8.9 on iPad Air 5th gen (running iPadOS 16.6.1).

**Comment by riverscuomo** [comment]
Well, cache will be a problem when the app is reloaded. But that doesn't explain immediate duplication.

**Comment by riverscuomo** [comment]
okay yeah wow this was a prob in my user.copywith method

```dart
// bundleIds: this.bundleIds + [constants.byThePeopleBundleId],
      // bundleDirectories: this.bundleDirectories,
      // playlistIds: this.playlistIds,
      bundleIds: bundleIds ?? this.bundleIds,
      bundleDirectories: bundleDirectories ?? this.bundleDirectories,
      playlistIds: playlistIds ?? this.playlistIds,

**Comment by riverscuomo** [comment]
still need to figure out cache which is not being updated.

**Comment by riverscuomo** [comment]
CacheHelper().savePlaylists(updatedAllPlaylists);

also managed to paint the blue border conditionally above or below drag target

**Comment by KCreek1** [comment]
This does not seem to be an issue in web app 1.9.0 (170).

**Comment by SlightlyCompletely** [comment]
Resolved for iPad Air 5th gen (running iPadOS 16.6.1) by update to Weezify v1.9.0 (170).
